src/TheThingsNetwork.cpp

```diff
@@ -283,8 +283,8 @@ int TheThingsNetwork::sendBytes(const byte* payload, int length, int port, bool
   }
 
   String response = "";
-  if (confirm == true) {
-    response = readLine(100000);
+  if (confirm) {
+    response = readLine(600000);
```

Comment:
This needs to be really indefinite. Just block. The RN module will respond eventually with `mac_tx_ok`, `mac_rx` or `mac_err`